meta-refkit/recipes-support/sbsigntool/sbsigntool-native_git.bb

```diff
@@ -0,0 +1,74 @@
+DESCRIPTION = "Utility for signing and verifying files for UEFI Secure Boot"
+LICENSE = "GPLv3 & LGPL-2.1 & LGPL-3.0 & MIT"
+
+# sbsigntool statically links to libccan.a which is built with modules
+# passed to "create-ccan-tree" (and their dependencies). Therefore,
+# we also keep track of all the ccan module licenses.
+LIC_FILES_CHKSUM = "file://LICENSE.GPLv3;md5=9eef91148a9b14ec7f9df333daebc746 \
+		    file://COPYING;md5=a7710ac18adec371b84a9594ed04fd20 \
```

Comment:
It is based on some early secureboot work (but not from meta-luv). I improved the recipe drastically so that we can not build it without any patching + we track ccan licenses.

I also share the same goal about the final location. Doing it in meta-refkit is temporary.